ci/test-in-cluster.sh

```diff
@@ -28,15 +26,8 @@ cp /secrets/hail-ci-0-1.key gcloud-token
 
 export IN_CLUSTER=true
 export SELF_HOSTNAME=https://ci.hail.is/$SERVICE_NAME
-export BATCH_SERVER_URL=http://127.0.0.1:5001
 export CLOUD_SQL_CONFIG_PATH=/batch-secrets/batch-test-cloud-sql-config.json
 
-pushd ../batch
-python3 -c 'import batch.server; batch.server.serve(5001)' & batch_pid=$!
```

Comment:
So in the cluster, we're using the existing batch server that's already running?

Reply:
So, previously, the local tests used an existing batch server (thus the batch proxy) and the PR tests used a batch server based on this branch's code. Now both local and PR tests use a local batch server. Since both use the local batch server, I start it in `test-locally.sh` which both environments use.